pkg/minikube/cluster/cluster.go

```diff
@@ -305,7 +305,7 @@ func createHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 	if err := api.Create(h); err != nil {
 		// Wait for all the logs to reach the client
 		time.Sleep(2 * time.Second)
-		return nil, fmt.Errorf("Error creating. %s", err)
+		return nil, fmt.Errorf("Error creating store. %s", err)
```

Comment:
This error text is right before another error text that talks about saving a store, I assumed that they are related.
If I'm wrong please let me know but we need to qualify the action, what are we creating here? Before the error was vague.